moveit_ros/hybrid_planning/local_planner/local_planner_component/src/local_planner_component.cpp

```diff
@@ -93,9 +93,13 @@ bool LocalPlannerComponent::initialize()
   }
 
   // Start state and scene monitors
+  // Set frequency faster than local planner frequency (If we don't call this it would default to 100Hz)
+  planning_scene_monitor_->setStateUpdateFrequency(2.0 * config_.local_planning_frequency);
```

Comment:
Based on the documentation:
```cpp
  /** @brief Update the scene using the monitored state. This function is automatically called when an update to the
     current state is received (if startStateMonitor() has been called).
      The updates are throttled to a maximum update frequency however, which is set by setStateUpdateFrequency(). */
  void updateSceneWithCurrentState();
```
I decided instead of calling `updateSceneWithCurrentState()` every time the local planning component gets invoked, I increased the state update frequency.

Reply:
Why a factor of 2? How about a factor of 1.1 so the updates are just barely faster than required.

Reply:
the documentation is misleading. The automated updates are throttled, but calling `updateSceneWithCurrentState()` will always immediately apply the latest incoming state of the CurrentStateMonitor to the scene. I think calling `updateSceneWithCurrentState()` is therefore better (more stable and more up-to-date) than running the throttled updates at an async rate. Having a factor of 1.1 (or even higher) may cause an artificial undersampling of the actual sensor input.